### ISSUE TITLE
Fluids do not show in multiblocks 1.16

### ISSUE THREAD

**Issue by lublak** [VazkiiMods/Patchouli#373]
Currently no fluids are rendered in multiblocks:
(but for the building counter, blocks left, it works)

Installed mods:
crafttweaker
quark
patchouli

![grafik](https://user-images.githubusercontent.com/44057030/110682637-7115b100-81db-11eb-8c9b-7d47189d94d7.png)

JSON:
```json
{
  "name": "Conduit",
  "icon": "minecraft:conduit",
  "category": "patchouli:vanilla",
  "pages": [
    {
      "type": "text",
      "text": "Conduit power!"
    },
    {
      "type": "multiblock",
      "multiblock": {
        "pattern": [
          [ "     ", "BBBBB", "     " ],
          [ " WWW ", "BWWWB", " WWW " ],
          [ " WWW ", "BWCWB", " WWW " ],
          [ " WWW ", "BWWWB", " WWW " ],
          [ "     ", "BB0BB", "     " ]
        ],
        "mapping": {
          " ": "ANY",
          "C": "minecraft:conduit",
          "B": "#forge:conduit_base_block",
          "0": "#forge:conduit_base_block",
          "W": "minecraft:water"
        },
        "symmetrical": true,
        "offset"     : [0, 0, 0],
        "view_offset": [0, 0, 0]
      },
      "text": "31 Block Range"
    }
  ]
}
```

Crafttweakerscript:
```
<tag:blocks:forge:conduit_base_block>.add([
	<block:minecraft:prismarine>,
	<block:minecraft:dark_prismarine>,
	<block:minecraft:prismarine_bricks>,
	<block:minecraft:sea_lantern>,
	<block:quark:elder_prismarine_bricks>,
	<block:quark:elder_prismarine>,
	<block:quark:dark_elder_prismarine>,
]);
```

**Comment by DaFuqs** [comment]
Confimed for Fabric 1.18

**Comment by williewillus** [comment]
Fixed by #552 (though in-world fluid render is a separate issue)
